ReactWindows/ReactNative.Shared/Bridge/IReactInstance.cs

```diff
@@ -26,11 +26,6 @@ public interface IReactInstance : IAsyncCancelable
         /// </summary>
         IReactQueueConfiguration QueueConfiguration { get; }
 
-        /// <summary>
```

Comment:
Can you just change the `Initialize` method on this interface to `InitializeAsync` for now? I suspect we'll be able to eliminate this interface all together soon (see #1899).

Reply:
Done